lua/neo-tree/sources/common/hijack_cursor.lua

```diff
@@ -0,0 +1,36 @@
+local events = require("neo-tree.events")
+local manager = require("neo-tree.sources.manager")
+local log = require("neo-tree.log")
+
+local M = {}
+
+local setup_for_module = function(module)
```

Comment:
I feel like this function name can be named better. Maybe something like `create_handler_for_module`?

Reply:
done